netcore/tests/OPADotNet.Benchmarks/OPADotNet.Benchmarks.csproj

```
@@ -2,11 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

```suggestion
		<TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Since we need to be backwards compatible